lib/linguist/heuristics.rb

```diff
@@ -313,6 +313,16 @@ def call(data)
       end
     end
 
+    disambiguate ".props" do |data|
+      if /^(\s*)(<Project|<Import|<Property|<\/Project|<\/Import|<\/Property)/i.match(data) 
+        Language["XML"]
+      elsif /\w+\s*=\s*/i.match(data)
+        Language["INI"]
+      else
```

Comment:
Hi, is there a suitable language with this kind of syntax:

```
key1 value1
key2     value2
```
(this is ini/config, but without `=`)

sure I will add a regex for it. :)